assets/js/blocks/cart-checkout/cart/full-cart/checkout-button/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import Button from '@woocommerce/base-components/button';
+import { CHECKOUT_URL } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { AmericanExpressLogo } from './american-express-logo';
+import './style.scss';
+import { MastercardLogo, VisaLogo } from './payment-logos'; // @todo we want to import this from `@automattic/composite-checkout` when it's published in NPM
```

Comment:
Agree. For extensibility, allowing extensions to hook into this space will probably be a requisite, so we might need to implement options 1 or 3. But agree this should be done in a future PR.

Reply:
Minor nit:  I prefer `@todo` added on the line above what it references so it's more visible.  No need to change if you disagree.

Reply:
I do prefer it too :smile: But in this case that was breaking the dependency lint rule. What that rule does is to check what the previous comment is, and since it was not `Internal dependencies` but `@todo ...` it was throwing a lint notice. We should improve that rule at some point, but for now I left the `@todo` comment inline so ESLint was not complaining.